docs/tools/vdb_table/data/couchbase.json

```diff
@@ -0,0 +1,181 @@
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "https://www.prnewswire.com/news-releases/couchbase-announces-new-features-to-accelerate-ai-powered-adaptive-applications-for-customers-302074941.html",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#hybrid",
+    "comment": ""
+  },
+  "facets": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/server/current/fts/fts-search-response-facets.html",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/search/geo-search-ui.html",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/create-vector-search-index-ui.html",
+    "comment": ""
+  },
+  "bm25": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "full_text": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/c-sdk/current/concept-docs/full-text-search-overview.html",
+    "comment": ""
+  },
+  "embeddings_text": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external embedding models"
+  },
+  "embeddings_image": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "embeddings_structured": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "rag": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external LLMs"
+  },
+  "recsys": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "langchain": {
+    "support": "full",
+    "source_url": "https://python.langchain.com/v0.1/docs/integrations/vectorstores/couchbase/",
+    "comment": ""
+  },
+  "llamaindex": {
+    "support": "full",
+    "source_url": "https://docs.llamaindex.ai/en/stable/api_reference/readers/couchbase/",
+    "comment": ""
+  },
+  "managed_cloud": {
+    "support": "full",
+    "source_url": "https://www.couchbase.com/products/capella/",
+    "comment": ""
+  },
+  "pricing": {
+    "value": "Couchbase Pricing",
+    "source_url": "https://www.couchbase.com/pricing",
+    "comment": ""
+  },
+  "in_process": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "multi_tenancy": {
+    "support": "full",
+    "source_url": "",
+    "comment": "via collections and RBAC"
+  },
+  "disk_index": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "ephemeral": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/buckets.html#replication-dcp-and-xdcr",
+    "comment": ""
```

Comment:
this column refers to the ability to have an index without a server (on client side itself)

Reply:
removing